src/adapters/supabase/helpers/client.ts

```diff
@@ -175,7 +175,7 @@ export const upsertWalletAddress = async (username: string, address: string): Pr
       created_at: new Date().toUTCString(),
       updated_at: new Date().toUTCString(),
     });
-    logger.info(`Creating a new wallet_table record done, { data: ${_data}, error: ${_error} }`);
+    logger.info(`Creating a new wallet_table record done, { error: ${_error?.message} }`);
```

Comment:
I'm assuming the issue is here. 
```suggestion
    logger.info(`Creating a new wallet_table record done, { data: ${_data}, error: ${_error?.message} }`);
```